File: packages/cli/templates/project/function-http.go
package main

import (
	"context"
	"encoding/json"
)

type HttpEvent struct {
	HTTPMethod string `json:"httpMethod"`
	Path       string `json:"path"`
}

type HttpResponse struct {
	StatusCode int               `json:"statusCode"`
	Headers    map[string]string `json:"headers,omitempty"`
	Body       string            `json:"body"`
}

func {{HANDLER}}(ctx context.Context, event HttpEvent) (HttpResponse, error) {
	requestID, _ := ctx.Value("lambdaRuntimeRequestID").(string)
	body, err := json.Marshal(map[string]any{
		"ok":        true,
		"method":    event.HTTPMethod,
		"path":      event.Path,
		"requestId": requestID,
	})
	if err != nil {
		return HttpResponse{}, err
	}
	return HttpResponse{
		StatusCode: 200,
		Headers:    map[string]string{"content-type": "application/json; charset=utf-8"},
		Body:       string(body),
	}, nil
}

File: packages/cli/templates/project/function-http.py
import json
from typing import Any


def {{HANDLER}}(event: dict[str, Any], context: Any) -> dict[str, Any]:
    return {
        "statusCode": 200,
        "headers": {"content-type": "application/json; charset=utf-8"},
        "body": json.dumps(
            {
                "ok": True,
                "method": event.get("httpMethod"),
                "path": event.get("path"),
                "requestId": getattr(context, "request_id", None),
            }
        ),
    }
